src/CoreShop/Bundle/FrontendBundle/Resources/install/pimcore/staticroutes.yml

```diff
@@ -247,17 +247,17 @@ routes:
     priority: 2
 
   coreshop_wishlist_summary:
-    pattern: '/(\w+)\/shop\/wishlist$/'
-    reverse: '/%_locale/shop/wishlist'
+    pattern: '/(\w+)\/shop\/wishlist(\/(.*))?$/'
+    reverse: '/%_locale/shop/wishlist/{%token}'
```

Comment:
Maybe `identifier` instead of `token`? Then it works for named lists too if we ever decide to implement #2147.